data/2018-10/twtalks.md

```diff
@@ -0,0 +1,35 @@
+---
+id: tw-th-20181003
+date: '2018-10-03'
+time: '18:00 ~ 21:00'
+location:
+  title: BIG Co-Working Space
+  url: https://www.google.com/maps/place/BIG+Co-working+Space/@13.7566648,100.5697955,17z/data=!3m1!4b1!4m5!3m4!1s0x30e29e8b209f69c3:0xf2124609ad0be030!8m2!3d13.7566648!4d100.5719842
+categories:
+  - Meetup
+topics:
+  - From the Iron Age to the Cloud Age by Keif Morris
+links:
+  - type: website
+    url: https://www.facebook.com/events/529475470810760/
+    title: Facebook Event
+  - type: rsvp
+    url: https://www.eventpop.me/e/4261-keifmorrisbkk
+    title: Event Pop
+---
+
+# ThoughtWorks Talks Tech: From the Iron Age to the Cloud Age by Keif Morris
+
+> ThoughtWorks Thailand is excited to welcome Keif Morris to Bangkok!
+
+Title: From the Iron Age to the Cloud Age by Keif Morris
+Abstract: The move to cloud is driving a shift in how we manage changes to infrastructure. Long cycles are out, and rapid improvement loops are in.  This talk describes the principles and practices that organisations and teams can apply to deliver more reliable infrastructure more quickly.
```

Comment:
It seems that the Markdown is not formatted nicely:

![image](https://user-images.githubusercontent.com/193136/46330839-32a95e80-c63f-11e8-9503-0866605f4454.png)

You can try using the https://calendar.thaiprogrammer.org/editor to preview how your data file would appear on the website.